easybuild/easyblocks/generic/pythonpackage.py

```diff
@@ -514,6 +516,16 @@ def compose_install_command(self, prefix, extrapath=None, installopts=None):
             },
         ])
 
+        ext_postinstallcmds = self.cfg['ext_postinstallcmds']
```

Comment:
Hmm, I wonder if it wouldn't have been better to use the existing `postinstallcmds` instead...

You can already specify `postinstallcmds` for a specific extension (just like any other common easyconfig parameter), but doing so won't have any effect, unless a change like this is made in easyblocks that are used to install extensions.

So, how about just renaming to `ext_postinstallcmds` to `postinstallcmds` here after all (and removing the support for picking up `ext_postinstallcmds`, while this is still only in `develop`?

cc @smoors, @ocaisa

Reply:
@boegel that would indeed be better.
I went for another parameter to avoid the clash with the global `postinstallcmds`, but it's easy to fix this, by adding this line to the __init__ function in extensionblock.py (similar to the `start_dir` parameter):
```
self.cfg['postinstallcmds'] = self.ext.get('options', {}).get('postinstallcmds', None)
```
because of the clash I assumed this was intended, but now I understand it was just not yet implemented.
I can make a PR for this if that's the way to go.